### ISSUE TITLE
[Bug]: fluent-accordion icons aligned incorrectly with multiline header

### ISSUE THREAD

**Issue by skansakar** [microsoft/fluentui#33347]
### Component

Accordion

### Package version

9.55.0

### React version

18.2.0

### Environment

```shell
-
```

### Current Behavior

When the accordion header is longer than one line, the icon is centred vertically. See screenshot from stackblitz. 
![Image](https://github.com/user-attachments/assets/2f66c551-ef04-4390-a750-5333c9b3ecd3)

### Expected Behavior

Expect the icon to be aligned vertically with the first line of text in the header

### Reproduction

https://stackblitz.com/edit/fgeddy?file=src%2Fexample.tsx

### Steps to reproduce

1. Create an accordion with a long header text that overflows to more than 1 line
2. Observe the icon's alignment issue.

### Are you reporting an Accessibility issue?

None

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by sopranopillow** [comment]
I agree, the chevron should be aligned at the top. As mentioned in the severity, this has a workaround by using a classname with makestyles. Thanks for reporting it @skansakar!
